### PR TITLE
Checks if user is in specified groups

### PR DESCRIPTION
**Pull request by marialivia16**

### CODE REVIEW

**Comment by jennysivapalan** [comment]
LGTM 👍

File: pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/service/Google2FAGroupChecker.scala [hunk 1]
```diff
@@ -55,7 +55,7 @@ class GoogleGroupChecker(config: Google2FAGroupSettings, bucket: S3Bucket) exten
 
   def checkGroups(authenticatedUser: AuthenticatedUser, groupIds: List[String]): Boolean = {
     val query = directory.groups().list().setUserKey(authenticatedUser.user.email)
-    if(groupIds.isEmpty) false else groupIds.foldLeft(true){(acc, groupId) => acc & hasGroup(query, groupId)}
```
Comment (jennysivapalan) [inline pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/service/Google2FAGroupChecker.scala]: Maybe worth logging the groupIds is empty?

Comment (marialivia16) [inline pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/service/Google2FAGroupChecker.scala]: Good idea. I wasn't sure what happens when we don't specify any groups. 

If the groupIds is empty we could also assume that we have no restrictions for the groups and this should return `true`.

**Comment by marialivia16** [comment]
@jennysivapalan I made the change we discussed, are you happy to merge this? 👀

**Comment by jennysivapalan** [comment]
👍

**Pull request merged**